### DISCUSSION TITLE
[ Implemented ]  Labled  Element and Constraint with custom name

### DISCUSSION THREAD

**Discussion by Opity69** [hlorus/CAD_Sketcher#79] [Feature Requests]
**Motivation / Existing Problem**
 unable to tell which Ratio constraint belongs to which  line  etc

**Description**
allow  for  naming  points and constraints with custom names( such and Core Box ratio constraint)

**Alternative Solutions**
N/A

**Mockup**

**Comment by hlorus** [comment]
This is partly planned for the next milestone, see: https://github.com/hlorus/CAD_Sketcher/issues/56. I'd say it's a higher priority for entities as they overlap more often.

Also note that elements get highlighted whenever the tooltip pops up.
